libsrc/effectengine/Effect.cpp

```diff
@@ -60,24 +61,8 @@ Effect::~Effect()
 	_imageStack.clear();
 }
 
-void Effect::run()
+void Effect::setModuleParameters()
```

Comment:
Content of this function also needs to be abstracted in PythonProgram. Let' do it one by one.